src/Tgstation.Server.Host/Extensions/ApplicationBuilderExtensions.cs

```diff
@@ -161,6 +162,13 @@ public static void UseServerBranding(this IApplicationBuilder applicationBuilder
 			applicationBuilder.Use((context, next) =>
 			{
 				context.Response.Headers.Add("X-Powered-By", assemblyInformationProvider.VersionPrefix);
+				if (context.Request.Method == HttpMethods.Put)
```

Comment:
This should be done in `ApiController.HookExecuteAction` not here. It's not what the function is documented to do and the scope of requests that go through here affect all HTTP requests, not just the REST API.